rules/windows/persistence_system_shells_via_services.toml

```diff
@@ -1,7 +1,8 @@
 [metadata]
 creation_date = "2020/02/18"
-maturity = "production"
-updated_date = "2021/03/03"
+deprecation_date = "2021/04/15"
+maturity = "deprecated"
```

Comment:
I've had some success with this rule in other environments but it's not perfect. Think there's anything redeemin if we narrowed the scope? Or is the noise still high volume and impossible to predict?

Reply:
This one isn't terrible, but hard to predict what some third parties will do that would generate a lot of noise, ie. NVIDIA. I'd be fine leaving this in prod for now and narrowing the scope down the road. I'll add some FP info in https://github.com/elastic/detection-rules/pull/1114/